content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Updating configuration via a ConfigMap
+content_type: tutorial
+---
+
+<!-- overview -->
+
+This page provides a step-by-step example updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. 
+
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+## {{% heading "objectives" %}}
+* Update configuration via a ConfigMap mounted as a Volume
+* Update environment variables of a Pod via a ConfigMap 
+
+<!-- lessoncontent -->
+
+## Update configuration via a ConfigMap mounted as a Volume
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap sport --from-literal=sport=football
+```
+
+[Create a Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment) manifest named `deployment.yaml` with the below specification. The ConfigMap `sport` is mounted as a Volume on the Pod.
```

Comment:
Ideally, make this an example that people can apply using `kubectl apply -f <url>`. See other tutorials to see how this is done, or ask questions if you're still not sure.

Reply:
So I can add a new yaml to the `content/en/examples directory`, right?

Reply:
You are welcome to do that.

Reply:
I guess I need some advice about this.
I added the yaml in the examples directory and it is rendered correctly.
`examples/deployments/deployment-with-configmap-as-volume.yaml`

Now as I understand, this yaml will be uploaded to the following location once the PR is merged to main, is that right?
https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/deployments/deployment-with-configmap-as-volume.yaml 

Currently this link throws a 404 as expected.

How do I validate that everything with this example yaml is correct before the PR is merged to main?

Reply:
This is resolved in the latest commit.